docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,53 +1,54 @@
 [[connect-to-elasticsearch]]
 == Add your data
 ++++
-<titleabbrev>Add data</titleabbrev>
+<titleabbrev>Add an integration</titleabbrev>
 ++++
 
-To start working with your data in {kib}, use one of the many ingest options,
-available from the home page.
-You can collect data from an app or service,
-or upload a file that contains your data.  If you're not ready to use your own data,
-add a sample data set and give {kib} a test drive.
+The best way to add data is to use one of our many integrations.
+Alternatively, you can add a sample data set or upload a file.
 
-[role="screenshot"]
-image::images/addData_home_7.15.0.png[Built-in options for adding data to Kibana:  Add data, Add Elastic Agent, Upload a file]
 
 [float]
 [[add-data-tutorial-kibana]]
-=== Add data
+=== Add an integration
 
-Want to ingest logs, metrics, security, or application data?
-Install and configure a Beats data shipper or other module to periodically collect the data
-and send it to {es}. You can then use the pre-built dashboards to explore and analyze the data.
+Elastic integrations are a streamlined way to connect your data to the Elastic Stack.
+Integrations are available for popular services and platforms, like Nginx, AWS, and MongoDB,
+as well as many generic input types like log files.
+Each integration comes pre-packaged with assets that support your needs:
 
-[role="screenshot"]
-image::images/add-data-tutorials.png[Add Data tutorials]
+* Data ingestion, storage, and transformation rules
+* Configuration options
+* Pre-built, custom dashboards and visualizations
+* Documentation
 
-[discrete]
-=== Add Elastic Agent
+To view the full list of available integrations, open the main
+menu and click *Add integrations*.
+{kib} automatically guides you to the *Integrations* view when no data is detected.
 
-*Elastic Agent* is the next generation of
-data integration modules, offering
-a centralized way to set up your integrations.
+[role="screenshot"]
+image::images/add-integration.png[Integrations page from which you can choose integrations to start collecting and analyzing data]
 
-With *Fleet*, you can add
-and manage integrations for popular services and platforms, providing
-an easy way to collect your data, and manage {elastic-agent} installations in standalone or {fleet} mode. The integrations
-ship with dashboards and visualizations,
-so you can quickly get insights into your data, and {fleet} mode offers several advantages:
+Use the controls on the bottom left of the **Integrations** view to determine whether to show
```

Comment:
Similar comments as above. While these are the two primary points of ingest, there are others. For this filter specifically, it may be worth discussing the logic. We default to Elastic Agent where the integration is GA, if it is not, we default to Beats integrations. for power users who know the shipper they want, they can use this filter to be explicit about what's shown.